src/main/java/org/apache/maven/plugins/artifact/buildinfo/CompareMojo.java

```diff
@@ -241,6 +241,11 @@ private void compareWithReference( Map<Artifact, String> artifacts, File referen
         }
 
         copyAggregateToRoot( buildcompare );
+
+        if ( ko + missing > 0 )
+        {
+            throw new MojoExecutionException( "Build artifacts are not reproducible" );
```

Comment:
s/not reproducible/different from reference/

they may be reproducible if rebuilder fixes his environment, for example uses the right JDK, or changes anything that is required

Reply:
fixed